frame/session/benchmarking/Cargo.toml

```diff
@@ -26,6 +26,7 @@ rand = { version = "0.7.2", default-features = false }
 [dev-dependencies]
 serde = { version = "1.0.126" }
 codec = { package = "parity-scale-codec", version = "2.0.0", features = ["derive"] }
+scale-info = { version = "0.9.0", default-features = false, features = ["derive"] }
```

Comment:
Missing `scale-info/std` feature for this Cargo.toml?

Reply:
Not required because a `dev-dependency`